Install/Chroot

```diff
@@ -223,12 +234,12 @@ Chroot images come with only one locale -- **en_US**, and use UTF-8 version of i
 If  you need to change it, type:
 
 {{{
-# cast -r locale
+# cast -r local
 }}}
 
-select those you want, and then -- change {{{LANG}}} variable in {{{/etc/sysconfig/locale}}} file to the one you'd like to use.
+Select those you want, and then change the {{{LANG}}} variable in {{{/etc/sysconfig/locale}}} file to the one you'd like to use.
 
-By default timezone is set to UTC. To change it, you need to choose timezone file and symlink it with {{{/etc/localtime}}}.
+By default timezone is set to UTC. To change itshutdow, you need to choose timezone file and symlink it with {{{/etc/localtime}}}.

```

Comment:
to change it… Or you can:
{{{
# cast -r tzdata
}}}

Reply:
I have corrected the typo. I have also added a small snippet around line 250 stating that an alternative method would be to re-cast or re-configure the tzdata. @vinpublic - the small code snippet that you included in your reply was included in the update.

This too should now be taken care of and should be fixed through the next pull request.